gpt_researcher/persistence/repository.py

```
@@ -0,0 +1,352 @@
+                batch = records[i:i + batch_size]
+                
+                # Generate embeddings concurrently for the batch
+                embedding_tasks = [
+                    self._get_embedding_async(record["raw_content"])
+                    for record in batch
+                ]
+                embeddings = await asyncio.gather(*embedding_tasks)
+                
+                # Prepare documents for insertion
+                documents = []
+                for record, embedding in zip(batch, embeddings):
```

Ruff (0.12.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)